apps/web/modules/ee/multi-language-surveys/components/default-language-select.tsx

```
@@ -30,11 +30,11 @@ export function DefaultLanguageSelect({
 }: DefaultLanguageSelectProps) {
   const { t } = useTranslate();
   return (
-    <div className="space-y-4">
-      <p className="text-sm">
-        {t("environments.surveys.edit.1_choose_the_default_language_for_this_survey")}:
-      </p>
-      <div className="flex items-center space-x-4">
+    <div className="space-y-2">
+      <label className="text-sm font-medium text-slate-800">
+        {t("environments.surveys.edit.1_choose_the_default_language_for_this_survey")}
+      </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)